test/GCClassic/parTestCreate.sh

```diff
@@ -0,0 +1,246 @@
+ln -s "${superProjectDir}" ${ptRoot}/CodeDir
+
+# Create log directory
+if [[ !(-d "${ptRoot}/logs") ]]; then
+    printf "\nCreating log directory: ${ptRoot}/logs\n"
+    mkdir "${ptRoot}/logs"
+fi
+
+# Log file with echoback from rundir creation
+log="${ptRoot}/logs/createParTests.log"
+
+# Switch to folder where rundir creation scripts live
+cd "${geosChemDir}/run/GCClassic"
+
+# Change to the directory where we will create the rundirs
+printf "\nCreating new run directories:\n"
+
+#=============================================================================
+# Create individual run directories: 4x5 - MERRA2 - 72L
+#=============================================================================
+
+# 4x5 merra2 CH4
+create_rundir "3\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 aerosol
+create_rundir "2\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# DEBUG: Exit after creating a couple of rundirs if $quick is "yes"
+if [[ "x${quick}" == "xyes" ]]; then
+    cd ${thisDir}
+    exit 0
+fi
+
+# 4x5 merra2 fullchem
+create_rundir "1\n1\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 fullchem_LuoWd
+dir="gc_4x5_merra2_fullchem_LuoWd"
+create_rundir "1\n1\n1\n1\n1\n${ptRoot}\n${dir}\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 fullchem_aciduptake
+create_rundir "1\n5\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 fullchem_APM
+create_rundir "1\n7\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 fullchem_benchmark
+create_rundir "1\n2\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 fullchem_complexSOA
+create_rundir "1\n3\n1\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 fullchem_complexSOA_SVPOA
+create_rundir "1\n3\n2\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 fullchem_marinePOA
+create_rundir "1\n4\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 fullchem_RRTMG
+create_rundir "1\n8\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 fullchem_TOMAS15_47L
+create_rundir "1\n6\n1\n1\n1\n2\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 Hg
+create_rundir "5\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 POPs_BaP
+create_rundir "6\n1\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 tagCH4
+create_rundir "7\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 tagCO
+create_rundir "8\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 tagO3
+create_rundir "9\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 TransportTracers
+create_rundir "10\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 TransportTracers_LuoWd
+dir="gc_4x5_merra2_TransportTracers_LuoWd"
+create_rundir "10\n1\n1\n1\n${ptRoot}\n${dir}\nn\n" "${log}" "${ptRoot}"
+
+# 4x5 merra2 metals"
+create_rundir "11\n1\n1\n1\n${ptRoot}\n\nn\n" "${log}" "${ptRoot}"
+
+##4x5 merra2 carboncycle -- COMING SOON
```

Comment:
This should be enabled now and renamed to carbon

Reply:
This is now done in commit 9f68416bf630f7e303e376f1868a0bd7121bcf53